[#497] Upload metadata and message source files (#602)
airyhq/airy: docs/docs/guides/airy-core-in-production.md

cp airy.conf.all airy.conf
```

Edit the file to configure connections to the base services. Make sure to configure the
following sections correctly, so that the `Airy apps` start properly:

```yaml
apps:
  kafka: ...
  redis: ...
  postgresql: ...
```

We recommend creating a new database if you are reusing a PostgreSQL server to avoid name collisions.

## Source media storage

Most message sources allow users to send rich data such as images, videos and audio files. For some sources
the Urls that host this data expire which is why after some time you may find that conversations have inaccessible
content.

The Airy Core Platform allows you to persist this data to a storage of your choice. To take advantage of this

> you need to provide access credentials to your storage. The platform currently only supports

Reviewer: ```suggestion
you must provide access credentials to your storage. The platform currently supports
```
